apollo-router/src/router_factory.rs

```diff
@@ -866,6 +894,8 @@ mod test {
                 Arc::new(schema),
                 None,
                 None,
+                // FIXME: fixture?
+                LicenseState::default(),
```

Comment:
__NIT__: for a lot of tests, you can simplify with `Default::default()` to avoid importing `LicenseState` everywhere.

Reply:
good call; will update